LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -6277,15 +6279,57 @@ ComIntLabels[eComInt_Savant]="Very High"
 ; End Translation
 
 ; LWOTC Needs Translation
-[ChosenPrimeReactionPassive X2AbilityTemplate]
+
+[AssassinPrimeReactionPassive X2AbilityTemplate]
+LocFriendlyName="Chosen Prime Reaction"
+LocLongDescription="Gain a slash, movement action or Combat Readiness after taking damage on XCOM's turn"
+LocHelpText="Gain a movement action or Combat Readiness after taking damage on XCOM's turn."
+LocFlyOverText="Chosen Prime Reaction"
+
+[HunterPrimeReactionPassive X2AbilityTemplate]
+LocFriendlyName="Chosen Prime Reaction"
+LocLongDescription="Gain a movement action or Combat Readiness after taking damage on XCOM's turn."
+LocHelpText="Gain a movement action or Combat Readiness after taking damage on XCOM's turn."
+LocFlyOverText="Chosen Prime Reaction"
+
+[HunterPrimeReactionPassive X2AbilityTemplate]
 LocFriendlyName="Chosen Prime Reaction"
 LocLongDescription="Gain a movement action or Combat Readiness after taking damage on XCOM's turn."
 LocHelpText="Gain a movement action or Combat Readiness after taking damage on XCOM's turn."
 LocFlyOverText="Chosen Prime Reaction"
 
+[ChosenImmunitiesPassive X2AbilityTemplate]
+LocFriendlyName="Chosen Immunities"
+LocLongDescription="Chosen are immune to stun, disorient, frost and suffer -50% mobility from maim instead of being immobilized."
+LocHelpText="Chosen are immune to stun, disorient, frost and suffer -50% mobility from maim instead of being immobilized."
+LocFlyOverText="Chosen Immunities"
```

Comment:
```suggestion
; LWOTC Needs Translation (2)
[ChosenImmunitiesPassive X2AbilityTemplate]
LocFriendlyName="Chosen Immunities"
LocLongDescription="Chosen are immune to stun, disorient, and frost and suffer -50% mobility from maim instead of being immobilized."
LocHelpText="Chosen are immune to stun, disorient, and frost and suffer -50% mobility from maim instead of being immobilized."
LocFlyOverText="Chosen Immunities"
; End Translation (2)
```